bundle/manifests/operator.istio.io_istios.yaml

```diff
@@ -146,15 +146,17 @@ spec:
                           labelPods:
                             type: object
                           tag:
-                            type: string
+                            type: object
```

Comment:
This is probably wrong, but is caused by the fact that `tag` is not defined as a string, but as a `google.protobuf.Value`:

https://github.com/istio/istio/blob/829b055e493511009d40e5f564c8d9ffca028917/operator/pkg/apis/istio/v1alpha1/values_types.proto#L51

Reply:
It looks like a `google.protobuf.Value` can be any type. I don't really know which OpenAPIv3 type fits here